addons/sale_pdf_quote_builder/views/quotation_document_views.xml

```diff
@@ -39,9 +39,10 @@
                             />
                         </group>
                     </group>
-                    <group string="Attached To" groups="base.group_no_one">
```

Comment:
maybe keep the form_field_ids below in debug mode only?

Reply:
True :smile: Good catch!